src/FixedDataTableBufferedRows.js

```diff
@@ -156,8 +156,8 @@ class FixedDataTableBufferedRows extends React.Component {
     FixedDataTableTranslateDOMPosition(style, 0, containerOffsetTop, false);
 
     // NOTE (pradeep): Sort the rows by row index so that they appear with the right order in the DOM (see #221)
-    const sortedRows = sortBy(this._staticRowArray, (row) =>
-      get(row, 'props.ariaRowIndex', Infinity)
+    const sortedRows = this._staticRowArray.sort(
+      (row) => row?.props?.ariaRowIndex || Infinity
```

Comment:
Grabbed along the way.
lodash's `sortBy` and `get` is a bit slower compared to native sort and optional chaining operations.